packages/ds-ext/src/duckDBStorage.ts

```diff
@@ -1,17 +1,17 @@
-import { GetLinkItemsParams, ObserverStorage } from '@data-story/core';
+import { GetLinkItemsParams, ItemValue, ObserverStorage } from '@data-story/core';
 import type { Database as DatabaseType} from 'duckdb-async';
+import { createDataStoryDBPath } from './commands/createDataStoryDBPath';
 export class DuckDBStorage implements ObserverStorage {
   private db: DatabaseType | null = null;
-  private dbPath: string;
 
-  constructor(dbPath: string) {
-    this.dbPath = dbPath;
+  constructor() {
     this.initDatabase();
```

Comment:
This expression is not awaited. Being inside constructor it is not possible. But, similar to `close` what do you think about having an ObserverStorage interface method `async init()`?

Reply:
I think we can keep the `init` as is for now, since `DiagramObserverStorage` doesn't require an explicit `init` method.